loom/script/runtime/lsLuaState.cpp

```diff
@@ -591,21 +620,38 @@ Assembly *LSLuaState::loadExecutableAssemblyBinary(const char *buffer, long buff
     lmCheck(headerBytes.readUnsignedInt() == LOOM_BINARY_VERSION_MINOR, "minor version mismatch");
     unsigned int sz = headerBytes.readUnsignedInt();
 
-    utByteArray bytes;
-    bytes.resize(sz);
+    utByteArray *bytes = lmNew(NULL) utByteArray();
```

Comment:
I think we didn't have to return the bytearray before, so we just used a local, but now loading the bytes and loading the assembly are separate operations, so we need a way to get the bytearray and just returning a local would copy it afaik?